docs/hardware.rst

```diff
@@ -317,24 +352,19 @@ before it can be used. We recommend:
     attempts to suspend. This has `since been fixed <https://communities.intel.com/message/432692#432692>`__
     in a BIOS update. See these `release notes <https://downloadmirror.intel.com/26263/eng/RY_0359_ReleaseNotes.pdf>`__ (PDF) for more details.
 
-Later NUC revisions (the NUC7 and NUC8 series) typically include onboard WiFi
-and Bluetooth, and may use an Ethernet chipset not supported by SecureDrop's
-custom kernel. We are investigating workarounds for both issues. If you are 
-having trouble sourcing the NUC5i5MYHE, please `contact us <https://securedrop.org/help/>`__
-for more information on how to safely configure and use more recent NUCs.  
-
-Mac Minis
-~~~~~~~~~
-
-.. caution:: We have previously recommended the 2014 Apple Mac Minis (part
-  number MGEM2) for installing SecureDrop. The 2018 Apple Mac Mini (part number
-  MRTR2 or MRTT2) is not a viable candidate for installing SecureDrop, due to
-  hardware support issues. The instructions below apply if you want to
-  (re-)install SecureDrop on the 2014 version.
+2014 Mac Minis
+~~~~~~~~~~~~~~
 
+We previously recommended the 2014 Apple Mac Minis (part number MGEM2)
+for installing SecureDrop. These will soon be officially obsolete. Unfortunately
```

Comment:
Perhaps link "officially obsolete" to https://support.apple.com/en-us/HT201624 to make it clearer what this means in the context of Apple hardware?

Reply:
done.